build/docker/kmesh_compatible.yaml

```diff
@@ -0,0 +1,101 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: kmesh-deploy
+  labels:
+    app: kmesh
+spec:
+  selector:
+    matchLabels:
+      app: kmesh
+  template:
+     metadata:
+       labels:
+         app: kmesh
+     spec:
+       volumes:
+         # use cgroup requires
+         - name: mnt
+           hostPath:
+             path: /mnt
+         # for eBPF program into the host machine
+         - name: sys-fs-bpf
+           hostPath:
+             path: /sys/fs/bpf
+         # required for compiling and building ko
+         - name: lib-modules
+           hostPath:
+             path: /lib/modules
+         # k8s default cni conflist path
+         - name: cni 
+           hostPath:
+             path: /etc/cni/net.d
+         # k8s deafult cni path 
+         - name: kmesh-cni-install-path
+           hostPath:
+             path: /opt/cni/bin
+         # Optional: 
+         # online compilation image needs to determine the support 
+         # to kmesh in host during compilation, based on this file.
+         - name: linux-bpf
+           hostPath:
+             path: /usr/include/linux/bpf.h
+         # Optional: 
+         # online compilation image needs compile kmesh.ko by host file
+         - name: ko-build-path
+           hostPath:
+             path: /usr/src
+       containers:
+         - name: kmesh
+           image: kmesh:latest
+           imagePullPolicy: IfNotPresent
+           command: ["/bin/sh", "-c"]
+           args: ["./start_kmesh.sh -enable-kmesh -enable-ads=true"]
+           securityContext:
+             privileged: true
+             capabilities:
+               add: ["all"]
+           ports:
+           - containerPort: 6789
+             hostPort: 6789
+           env:
+           - name: MESH_CONTROLLER
+             value: istio-system:istiod
```

Comment:
This is not right @YaoZengzeng

Reply:
we have another yaml, this yaml is used for compatible images for old users

Reply:
I donot think we need to, kmesh is very alpha. This is like a temporary workaround, we donot have to maintain two daemonset yamls